cpp/pybind/t/geometry/pointcloud.cpp

```diff
@@ -198,6 +198,18 @@ The attributes of the point cloud have different levels::
             },
             "Downsamples a point cloud with a specified voxel size.",
             "voxel_size"_a);
+    pointcloud.def(
+            "uniform_down_sample",
```

Comment:
You may use the following:
Instead of 
```cpp
pointcloud.def(
            "uniform_down_sample",
            [](const PointCloud& pointcloud, const size_t every_k_points) {
                return pointcloud.UniformDownSample(every_k_points);
            },
            "Downsamples a point cloud uniformly.", "every_k_points"_a);
  ```
  you may try:
  ```cpp
 pointcloud.def(
            "uniform_down_sample", &UniformDownSample,
            "Downsamples a point cloud uniformly.", "every_k_points"_a);
  ```
  
  similarly to other functions. 
  
  Also, test the same if it is working, you may find similar use in pybind folder for reference.

Reply:
Down